app/controllers/microservices_engine/v1/data_controller.rb

```diff
@@ -42,11 +42,11 @@ def update
         data = params['content']
         if data
```

Comment:
Yup, `present?` is a Activesupport thing (so [added by Rails](http://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark)).